src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/OpprettBehandling/OpprettBehandlingValg.tsx

```diff
@@ -193,53 +151,24 @@ const OpprettBehandlingValg: React.FC<IProps> = ({
-                                      (kanOppretteMigreringsbehandlingMedEndreMigreringsdato &&
-                                          årsak === BehandlingÅrsak.ENDRE_MIGRERINGSDATO)
-                              )
-                              .map(årsak => {
-                                  return (
-                                      <option
-                                          key={årsak}
-                                          aria-selected={behandlingsårsak.verdi === årsak}
-                                          value={årsak}
-                                      >
-                                          {behandlingÅrsak[årsak]}
-                                      </option>
-                                  );
-                              })
-                        : Object.values(BehandlingÅrsak)
-                              .filter(
-                                  årsak =>
-                                      årsak !== BehandlingÅrsak.TEKNISK_OPPHØR &&
-                                      årsak !== BehandlingÅrsak.TEKNISK_ENDRING &&
-                                      årsak !== BehandlingÅrsak.FØDSELSHENDELSE &&
-                                      årsak !== BehandlingÅrsak.SATSENDRING &&
-                                      årsak !== BehandlingÅrsak.MIGRERING &&
-                                      årsak !== BehandlingÅrsak.OMREGNING_6ÅR &&
-                                      årsak !== BehandlingÅrsak.OMREGNING_18ÅR &&
-                                      årsak !== BehandlingÅrsak.OMREGNING_SMÅBARNSTILLEGG &&
-                                      (årsak !== BehandlingÅrsak.KORREKSJON_VEDTAKSBREV ||
-                                          toggles[ToggleNavn.kanManueltKorrigereMedVedtaksbrev]) &&
-                                      årsak !== BehandlingÅrsak.ENDRE_MIGRERINGSDATO &&
-                                      årsak !== BehandlingÅrsak.HELMANUELL_MIGRERING
-                              )
-                              .map(årsak => {
-                                  return (
-                                      <option
-                                          key={årsak}
-                                          aria-selected={behandlingsårsak.verdi === årsak}
-                                          value={årsak}
-                                      >
-                                          {behandlingÅrsak[årsak]}
-                                      </option>
-                                  );
-                              })}
+                    {Object.values(BehandlingÅrsak)
+                        .filter(
+                            årsak =>
+                                årsak !== BehandlingÅrsak.SATSENDRING &&
+                                (årsak !== BehandlingÅrsak.KORREKSJON_VEDTAKSBREV ||
+                                    toggles[ToggleNavn.kanManueltKorrigereMedVedtaksbrev])
```

Comment:
Dette er bare halvferdig i ba-sak. Burde vi fjerne det helt og heller legge det til når det er ferdig? Ser for meg at det kan bli lett å glemme denne toggelen når den fjernes fra ba-sak

Reply:
Er det bare toggelen som skal fjernes her i såfall? Har ikke oversikt over hva dette gjør.

Slik tenker du?

`årsak =>
                                årsak !== BehandlingÅrsak.SATSENDRING &&
                                årsak !== BehandlingÅrsak.KORREKSJON_VEDTAKSBREV`

Reply:
Tar dette med i neste PR når det er avklart!